ext/reflection/php_reflection.c

```diff
@@ -4190,7 +4190,7 @@ ZEND_METHOD(ReflectionClass, getMethods)
 	reflection_object *intern;
 	zend_class_entry *ce;
 	zend_function *mptr;
-	zend_long filter = 0;
```

Comment:
I made these params nullable because the previous default value was horrible (and `ReflectionClass:getMethods()` was already nullable)